library/java/org/chromium/net/impl/CronetUrlRequestContext.java

```diff
@@ -327,7 +336,7 @@ private static void postObservationTaskToExecutor(Executor executor, Runnable ta
     try {
       executor.execute(task);
     } catch (RejectedExecutionException failException) {
-      // TODO(carloseltuerto): use Envoy-Mobile logs - this is a hack.
+      // TODO: use Envoy-Mobile logs - this is a hack.
```

Comment:
OOC, what would be required to use an EM log? Would you mind filing an issue about this and linking to it here?

Reply:
Well, I believe that the Java Logger is controlled by the C++ layer with Cronet. In fact, I am not sue at all how logging works with Cronet, and how it should work with EM.

Reply:
Please file an issue about this and link to it here.

Reply:
Issue filed: #2262